packages/terra-tabs/src/_CollapsibleTabs.jsx

```diff
@@ -120,7 +120,7 @@ class CollapsibleTabs extends React.Component {
       const tab = this.container.children[i];
       const tabMarginLeft = parseInt(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-left'), 10);
       const tabMarginRight = parseInt(window.getComputedStyle(this.menuRef, null).getPropertyValue('margin-right'), 10);
-      const minWidth = parseInt(window.getComputedStyle(tab, null).getPropertyValue('min-width'), 10);
+      const minWidth = parseFloat(window.getComputedStyle(tab, null).getPropertyValue('min-width'));
```

Comment:
Should we make this update on the margin values too?

Reply:
to keep change as much minimal I have not considered changing parse function for margin values. Yes it makes sense to update those values too : 38e8074fbcc168091ead784f2ef7721d16437378